src/config/dynamic-theme-fixes.config

```diff
@@ -6744,10 +6744,15 @@ IGNORE IMAGE ANALYSIS
 
 reddit.com
 
-INVERT
-#COIN_PURCHASE_DROPDOWN_ID > div > span
-
 CSS
+#SHORTCUT_FOCUSABLE_DIV > div:nth-child(3) > header > div > div._3dnbqz69WJTFCss8wl7Wlk > a > svg._1bWuGs_1sq4Pqy099x_yy- > g {
```

Comment:
`div._3dnbqz69WJTFCss8wl7Wlk`
`svg._1bWuGs_1sq4Pqy099x_yy-`
Needs to be replaced, with something else. The `_[...]` is random generated.

Reply:
It seems consistent for me across multiple webbrowsers and login-states. It doesn't change for me at all, maybe it's region depended?

Reply:
It's generated on production build by the front-end software Reddit uses. If they release an new 'production' build to reddit.com the CSS could break as the CSS is then invalid.